btclightclient/btc_light_client.go

```diff
@@ -225,11 +225,20 @@ func (lc *BTCLightClient) Status() {
 	fmt.Println(lc.btcStore.MostDifficultFork())
 }
 
+func (lc *BTCLightClient) LatestBlockHeight() int64 {
```

Comment:
This is not LatestBlockHeight. I think for right meaning this must be LatestFinalizedBlockHeight

Reply:
Then shall I also change the name in `Store` interface?

Reply:
Yes. Please!